app/presenters/decidim/calendar/event_presenter.rb

```diff
@@ -99,7 +99,9 @@ def all_day?
       def hour
         return nil if start.is_a?(Date) || all_day?
 
-        start.strftime("%H:%M")
+        hour12 = Decidim::Calendar.calendar_options[:hour12]
+
+        hour12 ? start.strftime("%I:%M %p") : start.strftime("%H:%M")
```

Comment:
```suggestion
        format = Decidim::Calendar.calendar_options[:hour12] ? "%I:%M %p" : "%H:%M"

        start.strftime(format)
```
Maybe it's a little bit cleaner this way?

Reply:
Thanks, done.